swagger.yaml

```diff
@@ -594,6 +594,9 @@ components:
           required:
             - participants
           properties:
+            number_of_results:
```

Comment:
This field should be required

Reply:
It's not required in any other draw, it just defaults to 1 if you don't provide it.